src/comodo/robotModel/robotModel.py

```diff
@@ -22,14 +23,28 @@ def __init__(
         left_foot: str = "l_sole",
         right_foot: str = "r_sole",
         torso: str = "chest",
+        right_foot_rear_link_name: str = "r_foot_rear",
+        right_foot_front_link_name: str = "r_foot_front",
+        left_foot_rear_link_name: str = "l_foot_rear",
+        left_foot_front_link_name: str = "l_foot_front",
+        kp_pos_control: np.float32 = np.array(
+            [35 * 70.0, 35 * 70.0, 35 * 40.0, 35 * 100.0, 35 * 100.0, 35 * 100.0,35 * 70.0, 35 * 70.0, 35 * 40.0, 35 * 100.0, 35 * 100.0, 35 * 100.0,20 * 5.745, 20 * 5.745, 20 * 5.745, 20 * 1.745,20 * 5.745, 20 * 5.745, 20 * 5.745, 20 * 1.745]
+        ),
+        kd_pos_control: np.float32= np.array([15 * 0.15, 15 * 0.15, 15 * 0.35, 15 * 0.15, 15 * 0.15, 15 * 0.15,15 * 0.15, 15 * 0.15, 15 * 0.35, 15 * 0.15, 15 * 0.15, 15 * 0.15,4 * 5.745, 4 * 5.745, 4 * 5.745, 4 * 1.745,4 * 5.745, 4 * 5.745, 4 * 5.745, 4 * 1.745])
```

Comment:
To be honest, this seems a bit ergoCub specific. Is this wanted?

Reply:
What happens if the foot is not split in two parts?

Reply:
I agree this could be improved (By passing a list of link names per foot). Anyhow, I have currently a few already implemented PR  to open and this change would be breaking. I prefer to implement this enhancement when I finalize the other review. I have opened an issue to track this: https://github.com/ami-iit/comodo/issues/14